runtime/acala/src/lib.rs

```diff
@@ -1560,7 +1560,7 @@ type RebaseTokens = orml_tokens::Combiner<
 	AccountId,
 	IsLiquidToken,
 	orml_tokens::Mapper<AccountId, Tokens, ConvertBalanceHoma, Balance, GetLiquidCurrencyId>,
```

Comment:
```suggestion
	orml_tokens::Mapper<AccountId, Currencies, ConvertBalanceHoma, Balance, GetLiquidCurrencyId>,
```

It's `Currencies` on mandala runtime